surya/schema.py

```diff
@@ -163,14 +163,14 @@ class OCRResult(BaseModel):
 class TextDetectionResult(BaseModel):
     bboxes: List[PolygonBox]
     vertical_lines: List[ColumnLine]
-    heatmap: Any
-    affinity_map: Any
+    heatmap: Optional[Any]
+    affinity_map: Optional[Any]
     image_bbox: List[float]
 
 
 class LayoutResult(BaseModel):
```

Comment:
This was a bug I originally introduced, but there is no heatmaps param here, so assigning to it will do nothing